_pages/policies/tech-policies/security-incidents.md

```diff
@@ -46,7 +46,7 @@ To report a security incident, follow *all* of the steps below:
 
 1. Following notification to GSA, the Incident Response team will contact you requesting more information. If the incident is related to cloud.gov, please ensure they CC the cloud.gov team (cloud-gov-support@gsa.gov), but try to drive as much of the conversation back to [#incident-response](https://gsa-tts.slack.com/messages/incident-response) in Slack as possible.
 
-1. If you cannot access email or Slack, please call the GSA IT Service Desk at 1-866-450-5250 and ask them to forward the relevant information to the addresses above. 
+1. If you cannot access email or Slack, please [contact the GSA IT Service Desk]({{ "/gsa-internal-tools/#it-service-desk" | relative_url }}) and ask them to forward the relevant information to the addresses above. 
```

Comment:
Given we still manage Slack user access we should break this out into its own item and make it emailto:devops@gsa.gov

Reply:
Let's revisit as part of https://github.com/18F/tts-tech-portfolio/issues/60.

Reply:
Let's revisit as part of https://github.com/18F/tts-tech-portfolio/issues/60.